bindings/java/mongocrypt/.evergreen/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Test the Java bindings for libmongocrypt
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+export JAVA_HOME="/opt/java/jdk8"
+
+if [ "Windows_NT" = "$OS" ]; then
+   export JAVA_HOME=/cygdrive/c/java/jdk8
+else
+   export JAVA_HOME=/opt/java/jdk8
+fi
+
+./gradlew -version
+
+./gradlew -Djna.library.path="../../../cmake-build" clean check
```

Comment:
Perhaps the fix for windows is just changing this path based on system, like Mark did in the test script:
https://github.com/10gen/libmongocrypt/blob/813131d4d15eec9a832ec14d573364c86fdb90ca/.evergreen/test.sh#L19-L22

So probably append `/Debug` to this path if this is Windows.

Reply:
I tried this earlier and it didn't work.

    Adding paths from jna.library.path: C:/data/mci/libmongocrypt/cmake-build/Debug
    Trying C:\data\mci\libmongocrypt\cmake-build\Debug\mongocrypt.dll

So I'm going to defer.